Analysis/ingrid/private/likelihood_utils.nim

```diff
@@ -72,6 +72,61 @@ proc getInterpolatedWideDf*(df: DataFrame, num = 1000): DataFrame =
     dfLoc["Variable"] = constantColumn(tup[0][1].toStr, dfLoc.len)
     result.add dfLoc
 
+proc morphToA*(df: DataFrame, lineEnergies: seq, energy: float, offset = 1): (Tensor[int], Tensor[float]) =
```

Comment:
If you just write `seq` as a type, it is an implicit generic. I.e. the compiler will generate one procedure for each type with which you call this procedure. Say you call it with a `lineEnergies` argument that is `seq[int]` and then later with a `seq[float]`, the compiler generates two procedures.

Better write:

```suggestion
proc morphToA*(df: DataFrame, lineEnergies: seq[float], energy: float, offset = 1): (Tensor[int], Tensor[float]) =
```